src/components/UserInfo/UserInfo.tsx

```diff
@@ -1,7 +1,16 @@
-// Don't forget to import the React library
+import React from 'react';
+import { User } from '../../types';
 
-// Create a `UserInfo` component accepting a `user` object and use it to render
-// a `todo.user` in the list with some styling. (Show at least a `name` and an
-// `email` of the `user`)
+interface Props {
+  user: User,
+}
 
-// Add a default export statement for UserInfo component to use it in the other files
+export const UserInfo: React.FC<Props> = ({ user }) => {
```

Comment:
why not just use `User` type and then de-structure it in props area?
```suggestion
export const UserInfo: React.FC<User> = ({ name, email, company }) => {
```